draft-jasdips-regext-rdap-rpki.md

```diff
@@ -976,109 +980,136 @@ Here is an elided example of a BGPSec router certificate object in RDAP:
+information for X.509 resource certificate objects with issuer matching the "CN=ISP-*" pattern:
 
 ```
-https://example.net/rdap/rpki1/bgpsec_router_certs?issuer=CN%3DISP-*
+https://example.net/rdap/rpki1/x509_resource_certs?issuer=CN%3DISP-*
 ```
 
-Searches for BGPSec router certificate information by certificate subject are specified using this form:
+Searches for X.509 resource certificate information by certificate subject are specified using this form:
 
-rpki1/bgpsec_router_certs?subject=ZZZZ
+rpki1/x509_resource_certs?subject=ZZZZ
 
-ZZZZ is a search pattern per [@!RFC9082, section 4.1], representing the "subject" property of a BGPSec Router
-Certificate object, as described in (#bgpsec_router_cert_object_class). The following URL would be used to find
-information for BGPSec router certificate objects with subject matching the "CN=ROUTER-ISP-*" pattern:
+ZZZZ is a search pattern per [@!RFC9082, section 4.1], representing the "subject" property of an X.509 resource
+Certificate object, as described in (#x509_resource_cert_object_class). The following URL would be used to find
+information for X.509 resource certificate objects with subject matching the "CN=BGPSEC-ROUTE*" pattern:
 
 ```
-https://example.net/rdap/rpki1/bgpsec_router_certs?subject=CN%3DROUTER-ISP-*
+https://example.net/rdap/rpki1/x509_resource_certs?subject=CN%3DBGPSEC-ROUTE*
 ```
 
-Searches for BGPSec router certificate information by subject key identifier are specified using this form:
+Searches for X.509 resource certificate information by subject key identifier are specified using this form:
 
-rpki1/bgpsec_router_certs?subjectKeyIdentifier=BBBB
+rpki1/x509_resource_certs?subjectKeyIdentifier=BBBB
 
-BBBB is a string representing the "subjectKeyIdentifier" property of a BGPSec router certificate object, as described in
-(#bgpsec_router_cert_object_class). The following URL would be used to find a BGPSec router certificate object with
+BBBB is a string representing the "subjectKeyIdentifier" property of an X.509 resource certificate object, as described
+in (#x509_resource_cert_object_class). The following URL would be used to find an X.509 resource certificate object with
 subject key identifier matching the "hOcGgxqXDa7mYv78fR+sGBKMtWJqItSLfaIYJDKYi8A=" string:
 
 ```
-https://example.net/rdap/rpki1/bgpsec_router_certs?subjectKeyIdentifier=hOcGgxqXDa7mYv78fR+sGBKMtWJqItSLfaIYJDKYi8A=
+https://example.net/rdap/rpki1/x509_resource_certs?subjectKeyIdentifier=hOcGgxqXDa7mYv78fR+sGBKMtWJqItSLfaIYJDKYi8A=
+```
+
+Searches for X.509 resource certificate information by an IP address are specified using this form:
+
+rpki1/x509_resource_certs?ip=CCCC
+
+CCCC is a string representing an IPv4 or IPv6 address. The following URL would be used to find information for X.509
+resource certificate objects with the "ips" member encompassing the "192.0.2.0" IP address:
+
+```
+https://example.net/rdap/rpki1/x509_resource_certs?ip=192.0.2.0
```

Comment:
Give an IPv6 example.

Reply:
OK. I have an IPv6 CIDR example later but could also do it here.